src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -1098,13 +1098,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     </PropertyGroup>
 
     <ItemGroup>
-      <SatelliteAssemblyReferences Include="@(ReferencePath)" Condition="'%(Filename)' == 'mscorlib' or '%(Filename)' == 'netstandard' or '%(Filename)' == 'System.Runtime' " />
+      <_SatelliteAssemblyReferences Remove="@(_SatelliteAssemblyReferences)" />
```

Comment:
This is good, but I think we'd also want to clear the collection out _after_ the Csc task so that after all the Csc invocations happen and we yield back to the rest of the build there are no longer any <_SatelliteAssemblyReferences> Items hanging around.

Reply:
Does that matter? Are you just thinking that would free up a little space, or is there a correctness aspect to that?

Reply:
I think Kalle's comment was about correctness, but since we're making the item private anyway it's less of a concern.